include/snowhouse/exceptions.h

```diff
@@ -85,6 +85,8 @@ namespace snowhouse
   SNOWHOUSE_TEMPVAR(storage).compiler_thinks_i_am_unused(); \
   bool SNOWHOUSE_TEMPVAR(wrong_exception) = false; \
   bool SNOWHOUSE_TEMPVAR(no_exception) = false; \
+  bool SNOWHOUSE_TEMPVAR(more_info) = true; \
+  std::string SNOWHOUSE_TEMPVAR(info_string) = ""; \
```

Comment:
The assignment `= ""` is not necessary for initialization. (Matter of taste probably.)

Reply:
I personally prefer it for clarity but I don't mind removing it.

Reply:
Yes, as I said, it's a matter of taste, and I clearly prefer using the default constructor over some other constructor if possible.

Reply:
Yup, changed that already.